src/test/java/com/google/firebase/remoteconfig/FirebaseRemoteConfigClientImplTest.java

```diff
@@ -803,6 +1008,24 @@ private void checkPutRequestHeader(HttpRequest request, String query, String ifM
     assertEquals(ifMatch, headers.getIfMatch());
   }
 
+  private void checkPostRequestHeader(HttpRequest request, String query) {
```

Comment:
It's not exactly a `query`, but any URL suffix. I'd suggest renaming the argument to reflect that.

Reply:
Good point! Thanks! Updated.